gtsam/symbolic/tests/testSymbolicBayesTree.cpp

```diff
@@ -730,13 +681,13 @@ TEST(SymbolicBayesTree, COLAMDvsMETIS) {
   // METIS
   {
     Ordering ordering = Ordering::Create(Ordering::METIS, sfg);
-// Linux and Mac split differently when using mettis
+// Linux and Mac split differently when using Metis
 #if defined(__APPLE__)
-    EXPECT(assert_equal(Ordering(list_of(5)(4)(2)(1)(0)(3)), ordering));
+    EXPECT(assert_equal(Ordering{5, 4, 2, 1, 0, 3}, ordering));
 #elif defined(_WIN32)
-    EXPECT(assert_equal(Ordering(list_of(4)(3)(1)(0)(5)(2)), ordering));
+    EXPECT(assert_equal(Ordering{4, 3, 1, 0, 5, 2}, ordering));
 #else
-    EXPECT(assert_equal(Ordering(list_of(3)(2)(5)(0)(4)(1)), ordering));
+    EXPECT(assert_equal(Ordering{3, 2, 0, 5, 4, 1}, ordering));
```

Comment:
This is the CI failure, you have a typo, orig is `(list_of(3)(2)(5)(0)(4)(1))`

Reply:
Thanks, just saw that as well!!!